test/unit/ansible_provider_test.rb

```diff
@@ -35,13 +35,13 @@ class AnsibleProviderTest < ActiveSupport::TestCase
       it 'generates secrets properly' do
         params = {
           'remote_execution_ssh_password' => 'password',
-          'remote_execution_sudo_password' => 'letmein'
+          'ansible_become_password' => 'password'
```

Comment:
why it matches the other password, it would rather not, for the test to catch if we are assigning them incorrectly.

Reply:
Didn't get your point, but do you mean that it should be ` 'remote_execution_effective_user_password'` instead of `'ansible_become_password'`?

Reply:
no, that's not what I mean.
We've kept the `remote_execution_ssh_password => 'password'` and we use `password` for `ansible_become_password` as well, we should be able to distinquish each of the value in asserts and now we are not.
But as I said, it's a nitpick :)